From d1b966de2b32f6152bc3b9c3f5d842ba12407a87 Mon Sep 17 00:00:00 2001
From: Dan Williams <dan.j.williams@intel.com>
Date: Wed, 5 Jan 2022 13:32:32 -0800
Subject: [PATCH 078/217] Documentation: Drop attrs.adoc include

In preparation for switching build systems, drop the attrs.adoc include for
communicating variables to asciidoc. Simply add the necessary variable
values to the invocation of the command using the --attribute argument.

Link: https://lore.kernel.org/r/164141835217.3990253.17678912974035740752.stgit@dwillia2-desk3.amr.corp.intel.com
Tested-by: Alison Schofield <alison.schofield@intel.com>
Tested-by: Vaibhav Jain <vaibhav@linux.ibm.com>
Signed-off-by: Dan Williams <dan.j.williams@intel.com>
Signed-off-by: Vishal Verma <vishal.l.verma@intel.com>
---
 .gitignore                                      |  1 -
 Documentation/daxctl/Makefile.am                | 17 +++++++----------
 .../daxctl/daxctl-reconfigure-device.txt        |  2 --
 Documentation/ndctl/Makefile.am                 | 17 +++++++----------
 Documentation/ndctl/intel-nvdimm-security.txt   |  2 --
 Documentation/ndctl/ndctl-load-keys.txt         |  2 --
 Documentation/ndctl/ndctl-monitor.txt           |  2 --
 Documentation/ndctl/ndctl-sanitize-dimm.txt     |  2 --
 Documentation/ndctl/ndctl-setup-passphrase.txt  |  2 --
 Documentation/ndctl/ndctl-update-passphrase.txt |  2 --
 10 files changed, 14 insertions(+), 35 deletions(-)

diff -up ndctl-71.1/.gitignore.orig ndctl-71.1/.gitignore
--- ndctl-71.1/.gitignore.orig	2022-10-07 15:58:15.663005697 -0400
+++ ndctl-71.1/.gitignore	2022-10-07 15:58:57.238147247 -0400
@@ -23,7 +23,6 @@ Documentation/daxctl/asciidoctor-extensi
 Documentation/ndctl/asciidoctor-extensions.rb
 Documentation/cxl/asciidoctor-extensions.rb
 Documentation/cxl/lib/asciidoctor-extensions.rb
-Documentation/ndctl/attrs.adoc
 .dirstamp
 daxctl/config.h
 daxctl/daxctl
diff -up ndctl-71.1/Documentation/ndctl/intel-nvdimm-security.txt.orig ndctl-71.1/Documentation/ndctl/intel-nvdimm-security.txt
--- ndctl-71.1/Documentation/ndctl/intel-nvdimm-security.txt.orig	2020-12-22 16:44:57.000000000 -0500
+++ ndctl-71.1/Documentation/ndctl/intel-nvdimm-security.txt	2022-10-07 15:59:06.192177733 -0400
@@ -1,7 +1,5 @@
 // SPDX-License-Identifier: GPL-2.0
 
-include::attrs.adoc[]
-
 THEORY OF OPERATION
 -------------------
 The Intel Device Specific Methods (DSM) specification v1.7 and v1.8 [1]
diff -up ndctl-71.1/Documentation/ndctl/ndctl-load-keys.txt.orig ndctl-71.1/Documentation/ndctl/ndctl-load-keys.txt
--- ndctl-71.1/Documentation/ndctl/ndctl-load-keys.txt.orig	2020-12-22 16:44:57.000000000 -0500
+++ ndctl-71.1/Documentation/ndctl/ndctl-load-keys.txt	2022-10-07 15:59:06.192177733 -0400
@@ -1,7 +1,5 @@
 // SPDX-License-Identifier: GPL-2.0
 
-include::attrs.adoc[]
-
 ndctl-load-keys(1)
 ==================
 
diff -up ndctl-71.1/Documentation/ndctl/ndctl-monitor.txt.orig ndctl-71.1/Documentation/ndctl/ndctl-monitor.txt
--- ndctl-71.1/Documentation/ndctl/ndctl-monitor.txt.orig	2020-12-22 16:44:57.000000000 -0500
+++ ndctl-71.1/Documentation/ndctl/ndctl-monitor.txt	2022-10-07 15:59:06.192177733 -0400
@@ -1,7 +1,5 @@
 // SPDX-License-Identifier: GPL-2.0
 
-include::attrs.adoc[]
-
 ndctl-monitor(1)
 ================
 
diff -up ndctl-71.1/Documentation/ndctl/ndctl-sanitize-dimm.txt.orig ndctl-71.1/Documentation/ndctl/ndctl-sanitize-dimm.txt
--- ndctl-71.1/Documentation/ndctl/ndctl-sanitize-dimm.txt.orig	2020-12-22 16:44:57.000000000 -0500
+++ ndctl-71.1/Documentation/ndctl/ndctl-sanitize-dimm.txt	2022-10-07 15:59:06.192177733 -0400
@@ -1,7 +1,5 @@
 // SPDX-License-Identifier: GPL-2.0
 
-include::attrs.adoc[]
-
 ndctl-sanitize-dimm(1)
 ======================
 
diff -up ndctl-71.1/Documentation/ndctl/ndctl-setup-passphrase.txt.orig ndctl-71.1/Documentation/ndctl/ndctl-setup-passphrase.txt
--- ndctl-71.1/Documentation/ndctl/ndctl-setup-passphrase.txt.orig	2020-12-22 16:44:57.000000000 -0500
+++ ndctl-71.1/Documentation/ndctl/ndctl-setup-passphrase.txt	2022-10-07 15:59:06.193177737 -0400
@@ -1,7 +1,5 @@
 // SPDX-License-Identifier: GPL-2.0
 
-include::attrs.adoc[]
-
 ndctl-setup-passphrase(1)
 =========================
 
diff -up ndctl-71.1/Documentation/ndctl/ndctl-update-passphrase.txt.orig ndctl-71.1/Documentation/ndctl/ndctl-update-passphrase.txt
--- ndctl-71.1/Documentation/ndctl/ndctl-update-passphrase.txt.orig	2020-12-22 16:44:57.000000000 -0500
+++ ndctl-71.1/Documentation/ndctl/ndctl-update-passphrase.txt	2022-10-07 15:59:06.193177737 -0400
@@ -1,7 +1,5 @@
 // SPDX-License-Identifier: GPL-2.0
 
-include::attrs.adoc[]
-
 ndctl-update-passphrase(1)
 ==========================
 
